packages/contracts-bedrock/contracts/echidna/FuzzResourceMetering.sol

```diff
@@ -128,30 +128,37 @@ contract EchidnaFuzzResourceMetering is ResourceMetering, StdUtils {
 
     function _burnInternal(uint64 _gasToBurn) private metered(_gasToBurn) {}
 
+    // solhint-disable-next-line func-name-mixedcase
```

Comment:
Is there a way to do this for the entire file instead of line by line?

Reply:
Talking to @smartcontracts we decided to just remove that rule from the solhint config, it's the kind of rule we're not going to unintentionally break anyways.